xml/admin_monitoring_alerting.xml

```diff
@@ -41,72 +40,53 @@
  </itemizedlist>
  <para>
   The &prometheus; configuration and <emphasis>scrape</emphasis> targets
-  (exporting daemons) are setup automatically by &deepsea;. &deepsea; also
+  (exporting daemons) are setup automatically by &cephadm;. &cephadm; also
   deploys a list of default alerts, for example <literal>health
   error</literal>, <literal>10% OSDs down</literal>, or <literal>pgs
   inactive</literal>.
  </para>
 
- <sect1 xml:id="pillar-variables">
-  <title>Pillar Variables</title>
-  <para>The &spillar; is a key-value store that provides information and
-    configuration values to minions. It is available to all minions, each
-    with differing content. The &spillar; is pre-populated with default
-    values and can be customized in two different ways:</para>
-   <itemizedlist>
-    <listitem>
-     <para>
-      <emphasis role="bold"><filename>/srv/pillar/ceph/stack/global.yml</filename></emphasis>:
-      to change pillar variables for all nodes.
-     </para>
-    </listitem>
-    <listitem>
+ <sect1 xml:id="monitoring-cephadm-config">
+   <title>Configuring Monitoring and Alerting</title>
+   <para>
+     &cephadm; uses a configuration file to deploy the relevant services
+     for monitoring and alerting. See <xref linkend="cephadm-service-and-placement-specs"/>
+     for more information on creating service and placement specifications.
+   </para>
+   <para>
+     For example:
+   </para>
+<screen>
+  ---
```

Comment:
Proposal: The configuration file could be named `monitoring.yaml` and its content could be looking like the following.

[config file]

To apply configuration file and deploy the configured service you'd need to issue `ceph orch apply -i monitoring.yaml`.